explorer/lib/explorer/contract_managers/aligned_layer_service_manager.ex

```diff
@@ -26,7 +26,7 @@ defmodule AlignedLayerServiceManager do
 
     :error ->
       raise(
-        "Config file not read successfully, did you run make create-env? If you did,\n make sure Alignedlayer config file is correctly stored"
+        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure Eigenlayer config file is correctly stored"
```

Comment:
No, here we are reading the AlignedLayer config file, reading its deployed addresses, not Eigen's.
```suggestion
        "Config file not read successfully, did you run make explorer_create_env? If you did,\n make sure AlignedLayer config file is correctly stored"
```

Reply:
Done